pkg/yurtmanager/controller/daemonpodupdater/config/constants.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import corev1 "k8s.io/api/core/v1"
+
+const (
+	// UpdateAnnotation is the annotation key used in DaemonSet spec to indicate
+	// which update strategy is selected. Currently, "OTA" and "AdvancedRollingUpdate" are supported.
+	UpdateAnnotation = "apps.openyurt.io/update-strategy"
+
+	// OTAUpdate set DaemonSet to over-the-air update mode.
+	// In daemonPodUpdater controller, we add PodNeedUpgrade condition to pods.
+	OTAUpdate = "OTA"
+	// AutoUpdate set DaemonSet to Auto update mode.
+	// In this mode, DaemonSet will keep updating even if there are not-ready nodes.
+	// For more details, see https://github.com/openyurtio/openyurt/pull/921.
+	AutoUpdate            = "Auto"
```

Comment:
please remove `AutoUpdate` definition and related code.

Reply:
done